docs/src/main/paradox/routing-dsl/index.md

```diff
@@ -69,6 +69,36 @@ the Routing DSL will look like:
 
 @@@
 
+## Dynamic Routing Example
```

Comment:
Would be good with an introductory paragraph saying that as the routes are evaluated for each request it is possible to make it change at runtime. However every access may happen on a separate thread so any shared mutable state must be thread safe.